agent/provider/src/market/provider_market.rs

```diff
@@ -136,6 +142,7 @@ impl ProviderMarket {
             negotiator: Arc::new(NegotiatorAddr::default()),
             config: Arc::new(config),
             subscriptions: HashMap::new(),
+            postponed_demands: Vec::new(),
```

Comment:
I would store this in `CompositeNegotiator`, but as I see there are reasons to have this here? Some would be lacking in negotiator?